README.md

```diff
@@ -78,7 +78,7 @@ require 'talon_one'
 TalonOne.configure do |config|
   # Configure the API host destination
   config.host = 'mycompany.talon.one'
-
+  config.scheme = 'https'
```

Comment:
Maybe we should also add here a tiny comment like with the other settings to explain "why"...